eli5/xgboost.py

```diff
@@ -413,3 +540,13 @@ def _missing_values_set_to_nan(values, missing_value, sparse_missing):
     if not np.isnan(missing_value):
         values[values == missing_value] = np.nan
     return values
+
+
+def shap_prediction_weights(booster, dmatrix, n_targets):
```

Comment:
What do you think about adding `_` to the name of this function to make it private?

Reply:
@ivanprado Yes. It should be a private function. Adding with the next commit. Thanks